voucher/utils.py

```diff
@@ -113,6 +114,22 @@ def get_eligible_coupon_choices(voucher):
     ]
     if course_matches and not eligible_choices:
         log.error("Found no valid coupons for matches for voucher %s", voucher.id)
+
+    if len(eligible_choices) > 1 and voucher.course_title_input:
+        eligible_choices_titles = [choice[1] for choice in eligible_choices]
+        best_match = difflib.get_close_matches(
+            voucher.course_title_input,
+            eligible_choices_titles,
+            len(eligible_choices_titles),
+            0,
+        )
```

Comment:
will get_close_matches always return one item?

Reply:
It will return all of the items in sorted orders with respect to the close match. The best-matched item will come first and no matched item in the end.